lib/telemetry_metrics/sum.ex

```diff
@@ -22,8 +22,8 @@ defmodule Telemetry.Metrics.Sum do
           event_name: :telemetry.event_name(),
           measurement: Metrics.measurement(),
           tags: Metrics.tags(),
-          tag_values: (:telemetry.event_metadata() -> :telemetry.event_metadata()),
-          keep: (:telemetry.event_metadata() -> boolean()),
+          tag_values: Metric.tag_values(),
```

Comment:
This should be be `Metrics` with an s. can you please adjust here and everywhere else accordingly? Thank you!

Reply:
Whoops! Fixed now.